src/quo2/components/notifications/activity_log/style.cljs

```diff
@@ -46,8 +46,11 @@
    :margin-left    8
    :color          colors/neutral-40})
 
-(def unread-dot
-  {:background-color colors/primary-50
+(defn unread-dot
+  [customization-color]
+  {:background-color (if customization-color
+                       (colors/custom-color customization-color 50)
```

Comment:
I would assume this should be `(colors/custom-color customization-color 60)` since it's in the dark mode